collectors/slurm-epilog/Cargo.toml

```diff
@@ -31,6 +31,7 @@ strip = true
 
 [dependencies]
 auditor = { path = "../../auditor", version = "0.4.0" }
+auditor-client = { path = "../../auditor-client" }
```

Comment:
Should we also specify a version here, similar to the line above where we declare the dependency to `auditor`?